experimental/packages/opentelemetry-sdk-metrics-base/src/view/Aggregation.ts

```diff
@@ -24,19 +28,91 @@ import { InstrumentDescriptor } from '../InstrumentDescriptor';
  * Aggregation provides a set of built-in aggregations via static methods.
  */
 export abstract class Aggregation {
-  // TODO: define the actual aggregator classes
-  abstract createAggregator(instrument: InstrumentDescriptor): unknown;
+  abstract createAggregator(instrument: InstrumentDescriptor): Aggregator<Maybe<Accumulation>>;
 
-  static None(): Aggregation {
-    return NONE_AGGREGATION;
+  static Drop(): Aggregation {
+    return DROP_AGGREGATION;
+  }
+
+  static Sum(): Aggregation {
+    return SUM_AGGREGATION;
+  }
+
+  static LastValue(): Aggregation {
+    return LAST_VALUE_AGGREGATION;
+  }
+
+  static Histogram(): Aggregation {
+    return HISTOGRAM_AGGREGATION;
+  }
+
+  static Default(): Aggregation {
+    return DEFAULT_AGGREGATION;
   }
 }
 
-export class NoneAggregation extends Aggregation {
+export class DropAggregation extends Aggregation {
```

Comment:
Can you please explain or document how `Aggregation`, `Aggregator`, and `Accumulation` are meant to work together?

Reply:
I add a short explanation in the OP. I'll add more descriptive comments in the codebase.